src/rules/prefer-importing-jest-globals.ts

```diff
@@ -0,0 +1,66 @@
+import globalsJson from '../globals.json';
+import { createRule, parseJestFnCall } from './utils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Prefer importing Jest globals',
+      recommended: false,
+    },
+    messages: {
+      preferImportingJestGlobal: `Import the following Jest functions from '@jest/globals': {{ jestFunctions }}`,
+    },
+    fixable: 'code',
+    type: 'problem',
+    schema: [],
+  },
+  defaultOptions: [],
+  create(context) {
+    const jestGlobalFunctions = Object.keys(globalsJson);
+    const importedJestFunctions: string[] = [];
+    const usedJestFunctions = new Set<string>();
+
+    return {
+      CallExpression(node) {
+        const jestFnCall = parseJestFnCall(node, context);
+
+        if (!jestFnCall) {
+          return;
+        }
+        if (
+          jestFnCall.head.type === 'import' &&
+          jestGlobalFunctions.includes(jestFnCall.name)
+        ) {
+          importedJestFunctions.push(jestFnCall.name);
+        }
+
+        usedJestFunctions.add(jestFnCall.name);
+      },
+      'Program:exit'() {
+        const jestFunctionsToImport = Array.from(usedJestFunctions).filter(
+          jestFunction => !importedJestFunctions.includes(jestFunction),
+        );
+
+        if (jestFunctionsToImport.length > 0) {
+          const node = context.getSourceCode().ast;
+          const jestFunctionsToImportFormatted =
+            jestFunctionsToImport.join(', ');
+
+          context.report({
+            node,
+            messageId: 'preferImportingJestGlobal',
+            data: { jestFunctions: jestFunctionsToImportFormatted },
+            fix(fixer) {
+              return fixer.insertTextBefore(
+                node,
+                `import { ${jestFunctionsToImportFormatted} } from '@jest/globals';\n`,
```

Comment:
this should be either `import` or `require` depending on what the rest of the file uses. If neither form is used, these should be suggestions for either form

Reply:
we also need to ensure this comes after `'use strict'` if present, otherwise it'll break the whole file (we don't need to worry about hashbangs because they're parsed as comments, so this'll already come after that)

Reply:
`import` / `require` / `use strict` statements are now properly handled.